core/server/admin/app.js

```diff
@@ -41,6 +41,9 @@ module.exports = function setupAdminApp() {
         {maxAge: utils.ONE_YEAR_MS, fallthrough: false}
     ));
 
+    // Service Worker for offline support
+    adminApp.get(/^\/(sw.js|sw-registration.js)$/, require('./serviceworker'));
```

Comment:
My understanding is that ServiceWorker scope is limited by the browser rather than Ember (service workers are completely separate scripts to the admin) and right now browsers require a URL relative to the location of the service worker script. We want to cache API requests within the `/ghost/*` scope so the service worker script needs to be loaded from `/ghost/` so we can use a relative scope of `./`